src/website/src/pages/ec/components/hero-banner/docs/usage.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+  A Hero banner displays a prominent message and related optional actions.
+  Banners are displayed on the homepage below the site header of EC sites. It
+  conveys an important message to your audience. Hero banners are a good way to
+  channel your visitors to complete a specific task (ex: register to an event,
+  apply for funding... ). The user can choose to ignore them or interact with
+  them at any time.
+</Paragraph>
```

Comment:
Please tak eout line 10-14

Reply:
The call to action is mandatory, also this is the main goal of a Hero banner "channel users to complete a specific task"